server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+
+- view `Account` page
+
+==== Edit Registered WebAuthn Authenticator
+
+A user can edit the following information :
+
+- Label (WebAuthn authenticator's label the user entered on registering it)
+
+A user carries out the following operations on <<_account-service, `User Account Service`>>  :
+
+- view `Account` page
+- edit the text in `Public Key Credential Label`
+- push `Save` button
+
+==== Delete Registered WebAuthn Authenticator
+
+An administrator can delete the users' registered WebAuthn authenticators.
+
+An Administrator carries out the following operations on `Admin Console` :
+
+- open `Users -> (user) -> Credentials` tab
+- on `Disable Credentials` area, add `webauthn` to `Disable Types`
+- push `Disable Credential Types` button
+
+==== Re-Register WebAuthn Authenticator
+
+A user can re-register their WebAuthn authenticator. Newly registered WebAuthn authenticator overrides the old registered one.
+
+At first, an administrator carries out the following operations on `Admin Console` to require the user to register their WebAuthn authenticator after their login :
+
+- open `Users -> (user) -> Details` tab
+- set `WebAuthn Register` on `Required User Actions`
+- push `Save` button
+
+After that, the user carries out the following operations :
+
+- open the login form
+- fill in items and push `Login` button
+- after logged in, the user's browser asks the user to register their WebAuthn authenticator
+- after successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label
+
+==== Configuration
+
+An administrator can configure WebAuthn related operations as `WebAuthn Policy` per realm.
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> WebAuthn Policy` tab
+- configure items and push `Save` button
+
+The configurable items and their description are shown just below.
+
+|===
+|Configuration|Description
+
+|Relying Party Entity Name
+|Human-readable server name as WebAuthn Relying Party. This is mandatory configuration item and applied to the operation of registering WebAuthn authenticator. The default setting is "keycloak".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dictionary-pkcredentialentity[WebAuthn Specification].
+ 
+|Signature Algorithms
+|It tells the WebAuthn authenticator which signature algorithms be used for the https://www.w3.org/TR/webauthn/#public-key-credential[Public Key Credential] that can be used to sign and verify the https://www.w3.org/TR/webauthn/#authentication-assertion[Authentication Assertion]. Multiple algorithms can be specified. If no algorithm is specified, https://tools.ietf.org/html/rfc8152#section-8.1[ES256] is adapted. The default setting is ES256. This is optional configuration item and applied to the operation of registering WebAuthn authenticator.
```

Comment:
instead of "that can be used to sign and verify"  try "for signing and verifying"

Reply:
Thank you. I'll fix this sentence as you suggested.